zebra/if_netlink.c

```diff
@@ -1251,6 +1260,16 @@ int netlink_link_change(struct sockaddr_nl *snl, struct nlmsghdr *h,
 void interface_list(struct zebra_ns *zns)
 {
 	interface_lookup_netlink(zns);
+	if (!local_linkinfo_presence) {
+		vrf_with_netns = 1;
+		zlog_info("Netlink: VRFLite not supported by Kernel. Switching to VRF Netns backend");
+	}
```

Comment:
I'm not sure if this is a good idea. I'd prefer to disable the VRF netns backend unless the user explicitly switch it on, even when running on older kernels.